### PR TITLE
[kuku] Add new port

### PR DESCRIPTION
**Pull request by kiromaru**
Add new port for kuku library:
https://github.com/microsoft/Kuku

- What does your PR fix?
This is a new port

- Which triplets are supported/not supported? Have you updated the CI baseline?
Tested x86-windows, x86-windows-static, x64-windows, x64-windows-static, x64-linux, x64-osx. Other triplets should be supported as well.
I have not updated the CI baseline.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

File: ports/kuku/CONTROL [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Source: kuku
+Version: 2.0
+Homepage: https://github.com/microsoft/Kuku
+Description: Kuku is a simple open-source (MIT licensed) cuckoo hashing library developed by the Cryptography and Privacy Research group at Microsoft.
+Default-Features:
```
Comment (PhoebeHui) [inline ports/kuku/CONTROL:5]:
```suggestion
```

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
Thanks for the PR! Please ignore the CI failures, it doesn't relate to this changes.

**Comment by kiromaru** [comment]
I've removed the empty Default-Features entry.

**Review by PhoebeHui — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**